package.json

```diff
@@ -13,34 +13,52 @@
   "unpkg": "dist/ionicons.js",
   "collection": "dist/collection/collection-manifest.json",
   "collection:main": "dist/collection/index.js",
+  "exports": {
+    ".": {
+      "types": "./dist/types/index.d.ts",
+      "default": "./dist/index.js",
+      "require": "./dist/index.cjs.js"
+    },
+    "./components": {
+      "types": "./dist/components/index.d.ts",
+      "import": "./dist/components/index.js"
+    },
+    "./icons": {
+      "types": "./icons/index.d.ts",
+      "import": "./icons/index.mjs",
+      "require": "./icons/index.js"
+    }
+  },
   "scripts": {
-    "compile.scripts": "tsc -p scripts/tsconfig.json",
-    "build": "npm run compile.scripts && npm run build.files && npm run build.component && npm run collection.copy",
-    "build.files": "node scripts/build.js",
+    "build": "run-s clean build.*",
```

Comment:
I'm getting an error when running this command:

```bash
> ionicons@7.3.1 build
> run-s clean build.*


> ionicons@7.3.1 clean
> rimraf dist components icons www

sh: rimraf: command not found
ERROR: "clean" exited with 127.
```

Reply:
I've defined `rimraf` as dev dependency. Mind running `npm install` and try again?